app/scripts/controllers/transactions/index.js

```diff
@@ -1457,8 +1457,8 @@ export default class TransactionController extends EventEmitter {
     });
   }
 
-  _getTransactionCompletionTime(submittedTime) {
-    return Math.round((Date.now() - submittedTime) / 1000).toString();
+  _getTransactionCompletionTime(time) {
```

Comment:
`metricsParams.completion_time` is meant to track the amount of time it takes for a transaction to go from being in the submitted state to being confirmed. This change would instead make it simply track the time the transaction was created.

Reply:
yes